packages/ag-charts-website/src/features/api-documentation/components/SearchBox.tsx

```diff
@@ -80,6 +80,18 @@ export function SearchBox({
                     </div>
                 </div>
             )}
+
+            {data.length === 0 && (
+                <div className={styles.searchDropdown} onMouseDown={(e) => e.preventDefault()}>
+                    <div className={styles.searchOptions}>
+                        <div className={styles.searchOption}>
+                            <span className="font-size-small">
+                                We couldn't find any matches for "<b>{searchQuery}</b>"
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            )}
```

Comment:
I can't so a multiline suggestion for some reason...
I think you can simplify it by doing this:
```
            {searchQuery.length > 0 && inFocus && (
                <div className={styles.searchDropdown} onMouseDown={(e) => e.preventDefault()}>
                    <div className={styles.searchOptions}>
                        {data.length ? data.map((data, index) => (
                            <div
                                key={data.label}
                                ref={
                                    index === selectedIndex
                                        ? (ref) => ref?.scrollIntoView({ block: 'nearest', inline: 'start' })
                                        : null
                                }
                                className={classnames(styles.searchOption, {
                                    [styles.selected]: index === selectedIndex,
                                })}
                                onClick={() => handleClick(data)}
                                onMouseEnter={() => setSelectedIndex(index)}
                            >
                                {markResults && searchQuery ? (
                                    <Markdown>{data.label.replace(searchRegexp, `**$1**`)}</Markdown>
                                ) : (
                                    data.label
                                )}
                            </div>
                        )) : (
                            <span className="font-size-small">
                                We couldn't find any matches for "<b>{searchQuery}</b>"
                            </span>
                        )}
                    </div>
                </div>
            )}
```

Reply:
This looks good. I'll update.